src/Distributions.jl

```diff
@@ -7,9 +7,10 @@ using StatsFuns
 using StatsBase
 using Compat
 
+using Base.Random
```

Comment:
Yeah, but we're explicitly doing a `import Base: ... rand, rand!` below. Could just add an explicit `import Base.Random: GLOBAL_RNG`.